manticore/platforms/evm.py

```diff
@@ -2236,7 +2327,9 @@ def SELFDESTRUCT(self, recipient):
         self.suicide.add(address)
         prev_vm = self._pop_vm(rollback=False)
         if self.depth == 0:
-            self.last_pc = prev_vm.pc
+            tx = self._transactions[-1]
+            tx.last_pc = prev_vm.pc
+            tx.result_type = 'SELFDESTRUCT'
```

Comment:
tx.result_type doesn't exist?

Reply:
No.  It's just .result now. 
That fix sound familiar. .. checking.